masonry/src/event.rs

```diff
@@ -313,6 +312,7 @@ pub enum Update {
 }
 
 impl PointerEvent {
+    /// Create a [`PointerEvent::PointerLeave`] event with dummy values.
     pub fn new_pointer_leave() -> Self {
```

Comment:
Maybe this method should be at the bottom, as something most users won't need (which users would need this?)